src/V3EmitCImp.cpp

```diff
@@ -522,9 +526,10 @@ class EmitCImp final : EmitCFunc {
         EmitCFunc::visit(nodep);
     }
 
-    explicit EmitCImp(const AstNodeModule* modp, bool slow)
+    explicit EmitCImp(const AstNodeModule* modp, bool slow, std::deque<AstCFile*>& cfiles)
```

Comment:
In function arguments typically please add "r" suffix if there's a non-constant reference so is obvious will be modified.

Can do this to to members, m_cfiles -> m_cfilesr also.  That rule is sometimes broken when it's obvious what's going on.



```suggestion
    explicit EmitCImp(const AstNodeModule* modp, bool slow, std::deque<AstCFile*>& cfilesr)
```

Reply:
Changed.

Reply:
Instead of a 'Fixed' comment can you please use the Resolve button below on each?  Then it is easy to see if everything has been addressed. Thanks!